### ISSUE TITLE
Unable to see the end of the screen in my app 

### ISSUE THREAD

**Issue by nihp** [facebook/react-native#26537]
In my app I can't able to see the end of the screen. So I have added margin bottom for all the pages. Can anyone facing this same in your apps.

Here I am facing the same in all the screens in my app. So temporarily I have added marginBottom values. But it is not fair in all modal devices. 

Any solution for this. 

I have added like this in all my screens at the end.

`<View style = {{marginBottom:"70%"}}/>`

**react-native-cli**: 2.0.1
**react-native**: 0.58.4

**Comment by nihp** [comment]
My Expected behavior is not to add the marginBottom values to see the end of the screens

Actual behavior is now I am adding marginBottom values to see the end of the screens
  ```
  <KeyboardAvoidingView>
           <ScrollView>
           Here I have added all the fields in my screen. Below this, I have added margin bottom to see the entire fields in the screen
          <View style = {{marginBottom:"40%"}}/>
          </ScrollView>
   </KeyboardAvoidingView>
```

**Comment by sauzy34** [comment]
Give `ScrollView` `paddingBottom: {value}`, then you don't need to add additional <View/> node in the tree

**Comment by nihp** [comment]
I have a clarification question here. If the paddingBottom value is set as
10 or 20. Then if we move to other lower version devices it will shows a
lot of space below the screen. I have experienced this while adding
marginBottom values in the end.

**Comment by nihp** [comment]
I have a clarification question here. If the paddingBottom value is set as
10 or 20. Then if we move to other lower version devices it will show a
lot of space below the screen. 

I have experienced this while adding
marginBottom values in the end.

Also, I have removed the ScrollView and I am using KeyboardAwareScrollView

```
<KeyboardAvoidingView>
         Here I have added all the fields in my screen. Below this, I have added margin bottom to see the entire fields in the screen
 </KeyboardAvoidingView>
```

**Comment by sauzy34** [comment]
Is it happening in only latest react-native version?

**Comment by rockfight** [comment]
I am having the same issue but only after upgrading to 0.61.3. The bottom components are going below the screen for some reason. If I reload the app 1 in 10 time the components come to place but again it goes down on next reload.

**Comment by nihp** [comment]
@sauzy34 I am using the **react-native: 0.58.4** version

**Comment by hramos** [comment]
Please make sure to fill in the issue template if you decide to file a new issue.
